odoo/modules/migration.py

```diff
@@ -112,6 +119,7 @@ def _get_migration_files(pkg, version, stage):
             mapping = {
                 'module': opj(pkg.name, 'migrations'),
                 'maintenance': opj('base', 'maintenance', 'migrations', pkg.name),
+                'migrations': opj(tools.config['migrations_path'], pkg.name),
```

Comment:
for master only: support 'upgrades' along with 'migrations' for the `module` and `maintenance` mappings, for consistency